pages/supervisor-resources/new-supervisor-playbook/play-1.md

```diff
@@ -0,0 +1,100 @@
+power and represent the organization.
+
+This new power carries many nuances with it. For example, you may find
+resistance from others on your team who may have applied to the role and didn’t
+get selected and/or you may find resistance from team members who are no longer
+your peers and are now your direct reports.
+
+Furthermore, you will also start finding yourself in different conversations
+that make you privy to more sensitive information that cannot just be shared out
+as is. **You will have to make the determination of what is appropriate to
+filter out and share depending on the audience** (e.g., your peers, your team
+members, external parties, etc.). This can be daunting at first, and early on it
+may be helpful to explicitly ask your supervisor or work with them to understand
+what is appropriate to disseminate to others and what is to be kept
+confidential. Some of these new conversations might also not add value to your
+team by sharing with them. It’s helpful to check with your audience to know what
+does and does not feel useful to share.
+
+To balance this new power, it is important to focus on your soft skills. No
+longer is your technical or subject matter expertise the most important skill
+you have - **it’s the soft skills that make the difference, and you’ll need to
+rely on and refine these soft skills over time to minimize power struggles and
+conflict**. Strong communication; coaching and mentoring others; holding others
+accountable while remaining empathetic; becoming a champion and advocate for
+those that report to you; and being fair and equitable, among other things, are
+the critical skills that will help you succeed in the role. Remember, your
+primary focus is now on building a team, cultivating a healthy team and work
+culture, and ensuring that the team achieves its goals!
+
+## Action Items
+
+- [ ] Upon starting your new role, stop doing your old job:
+  - [ ] Work with your supervisor to come up with a transition plan to account
+        for anything that is currently ongoing or needs to be picked up by
+        someone else
+  - [ ] You may very well be tasked with helping backfill your previous role;
+        work with your supervisor to do this, especially if you’re new to things
+        like hiring and interviewing
+- [ ] Becoming a supervisor for the first time is a career change; work with
+      your supervisor to develop a professional growth plan to build and refine
+      soft skills and emotional intelligence in this new role
+- [ ] Manage your own emotions and reactions: stop and think before engaging
+      with others in any communication medium to consider both the intent of
+      what you’re about to put in the world and how it may be received and
+      perceived
+  - [ ] Ask yourself questions such as “what is my intent here?” or “how might
+        others perceive or interpret this?” to help determine what is best
```

Comment:
I noticed that there was some spacing between list items in play two that made it a bit easier on the eyes when viewing the pages, so this is a suggestion for adding spaces between other list items!

```suggestion
- [ ] Upon starting your new role, stop doing your old job:

  - [ ] Work with your supervisor to come up with a transition plan to account
        for anything that is currently ongoing or needs to be picked up by
        someone else
        
  - [ ] You may very well be tasked with helping backfill your previous role;
        work with your supervisor to do this, especially if you’re new to things
        like hiring and interviewing
        
- [ ] Becoming a supervisor for the first time is a career change; work with
      your supervisor to develop a professional growth plan to build and refine
      soft skills and emotional intelligence in this new role
      
- [ ] Manage your own emotions and reactions: stop and think before engaging
      with others in any communication medium to consider both the intent of
      what you’re about to put in the world and how it may be received and
      perceived
      
  - [ ] Ask yourself questions such as “what is my intent here?” or “how might
        others perceive or interpret this?” to help determine what is best
```